.editorconfig

```diff
@@ -0,0 +1,11 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+
+# 2 space indentation
+[*.java]
```

Comment:
this is nowhere near complete - I think it's misleading. We use the [Google Java Style](https://google.github.io/styleguide/javaguide.html) which is pretty comprehensive.

Reply:
True that, removed 👍